Description: Change enna_thumb path from /usr/bin to /usr/lib/enna to comply
  with FHS.
Author: Davide Cavalca <davide@geexbox.org>
Forwarded: no

Index: enna-0.4.0/src/bin/thumb.c
===================================================================
--- enna-0.4.0.orig/src/bin/thumb.c	2010-01-05 09:19:50.000000000 +0100
+++ enna-0.4.0/src/bin/thumb.c	2010-01-05 09:19:54.000000000 +0100
@@ -207,7 +207,7 @@
         {
             Ecore_Exe *exe;
 
-            snprintf(buf, sizeof(buf), PACKAGE_BIN_DIR"/enna_thumb --nice=%d", 19);
+            snprintf(buf, sizeof(buf), PACKAGE_LIB_DIR"/enna/enna_thumb --nice=%d", 19);
             exe = ecore_exe_run(buf, NULL);
             _thumbnailers_exe = eina_list_append(_thumbnailers_exe, exe);
         }
